codegen/csg.adoc

```diff
@@ -8,4 +8,193 @@ permalink: /codegen_custom_swagger_gentemplate/
 :RAS: RepreZen API Studio 
```

Comment:
Or maybe `custom-scg.adoc`.  Doesn't matter that much, just helps to be able to look at the filename and know what it is.